include/bluetooth/gatt.h

```diff
@@ -1419,6 +1419,9 @@ struct bt_gatt_read_params {
 			const struct bt_uuid *uuid;
 		} by_uuid;
 	};
+#ifdef CONFIG_BT_EATT
+	enum bt_att_bearer_option bearer_option;
+#endif /* CONFIG_BT_EATT */
```

Comment:
Only added the option to the read params so far, but it should work the same for the other actions.